_topic_map.yml

```
@@ -221,16 +232,20 @@ Topics:
 ---
 Name: Support
 Dir: support
-Distros: openshift-enterprise,openshift-webscale,openshift-online,openshift-dedicated
+Distros: openshift-enterprise,openshift-webscale,openshift-online,openshift-dedicated,openshift-origin
 Topics:
 - Name: Getting support
   File: getting-support
+  Distros: openshift-enterprise,openshift-dedicated
 - Name: Gathering data about your cluster
   File: gathering-cluster-data
-  Distros: openshift-enterprise,openshift-webscale,openshift-dedicated
+  Distros: openshift-enterprise,openshift-webscale,openshift-dedicated,openshift-origin
+- Name: Remote health monitoring with connected clusters
+  Dir: remote_health_monitoring
+  Distros: openshift-enterprise,openshift-webscale,openshift-dedicated,openshift-origin
 - Name: Remote health monitoring with connected clusters
   Dir: remote_health_monitoring
-  Distros: openshift-enterprise,openshift-webscale,openshift-dedicated
+  Distros: openshift-enterprise,openshift-dedicated,,openshift-webscale,openshift-origin
```

```suggestion
  Distros: openshift-enterprise,openshift-dedicated,openshift-webscale,openshift-origin
```
extra comma  - not sure if this is critical